### DISCUSSION TITLE
ROG Ally -  Fan Curves Don't Work?

### DISCUSSION THREAD

**Discussion by mufaaf** [seerge/g-helper#2963] [Q&A] [answered]
Installed Ghelper today on the ROG Ally, and it's a lovely piece of software.

I am having some problem making the fan curve work.
![IMG_0427](https://github.com/user-attachments/assets/976284c4-db8b-448e-a123-4adcd00d2a62)

As you can see from the image above, the GPU fan curve is set to 70C, 5600RPM, however on the right-hand side menu, it indicates the GPU fan speed is set to 2800RPM. The same goes for CPU. Both the fan curves are locked to 4200RPM and 2800RPM, no matter the temp. I don't mind the noise level as long as the internals are not cooking.

The RPM only ramps up when I click "Apply Custom Fan Curve", which I have seen a YouTuber says uses the fan curves set in the Asus Armoury Crate and not the ones set in GHelper.

Am I missing something here? I hope someone can help me out here. I would like to use the fan curves on GHelper and customize it till I find a sweet spot, but it doesn't seem to be working and I really don't want to touch 83C while playing.

**Comment by seerge** [comment]
@mufaaf hello, **[default modes along with their fan curves are baked in BIOS](https://github.com/seerge/g-helper?tab=readme-ov-file#rocket-performance-modes)**. They are same with AC. 

On your screenshot you don't have any custom fan curves applied, you just use a Balanced profile so fan chart on the left has no effect. 

If you want fans behave more aggressively - use Turbo profile as a base, or apply custom fan curve. But please keep in mind, even if you will use custom fan curves - on Ally BIOS heavily overrides any custom fan curves (the same would happen in AC's manual mode).

**Reply by mufaaf** [reply]
> @mufaaf hello, **[default modes along with their fan curves are baked in BIOS](https://github.com/seerge/g-helper?tab=readme-ov-file#rocket-performance-modes)**. They are same with AC.
> 
> On your screenshot you don't have any custom fan curves applied, you just use a Balanced profile so fan chart on the left has no effect.
> 
> If you want fans behave more aggressively - use Turbo profile as a base, or apply custom fan curve. But please keep in mind, even if you will use custom fan curves - on Ally BIOS heavily overrides any custom fan curves (the same would happen in AC's manual mode).

Hey!

So if I am understanding you correctly, I have to click "Apple Custom Fan Curve" on my custom TDP profile (25W, as show in the image) for the fan curves I have set on GHelper to kick in? I did try it but the fans just go up to 6000RPM+ and doesn't adhere to the levels set on GHelper.

So if Asus overrides any custom fan curves, that makes the fan curve option in GHelper useless? 

I am not sure if I truly understand the problem, sorry :p

**Reply by seerge** [reply, marked as answer]
@mufaaf if you want to use custom fan curve you need to click `Apply` of course. But BIOS can override custom fan curves if it believes that temps are too high for that fan speeds, **same as it does it when you set them in Armoury.**

That's why some people on reddit just suggest to pick one of 3 default modes (along their default fan curves) and customize only power limits to achieve their goals.

On your screenshot you use Balanced mode (that by default comes with 15w limits) and set 25w limits there. If you want fans behave more aggressively you can use Turbo as a base and customize it in turn

**Reply by mufaaf** [reply]
> @mufaaf if you want to use custom fan curve you need to click `Apply` of course. But BIOS can override custom fan curves if it believes that temps are too high for that fan speeds, **same as it does it when you set them in Armoury.**
> 
> That's why some people on reddit just suggest to pick one of 3 default modes (along their default fan curves) and customize only power limits to achieve their goals.
> 
> On your screenshot you use Balanced mode (that by default comes with 15w limits) and set 25w limits there. If you want fans behave more aggressively you can use Turbo as a base and customize it in turn

Ahh, I see what you mean. I used the Turbo profile and created a custom 25W profile on top of it and now the fans adhere to the set fan curve on GHelper better and more accurately. Fair enough. That solves the issue! Thanks!

**Reply by seerge** [reply]
@mufaaf alternatively you can use [Experimental build with manual fan control](https://github.com/seerge/g-helper/discussions/2272). This is the only case where G-Helper will actually try to set fan speeds in real-time :) But only if default fan behavior bothers you a lot
